cg-spec/editors_draft.html

```diff
@@ -2680,7 +2705,7 @@ <h2>Referential opacity</h2>
         -->
         </pre>
 
-        <p>Similarly, in the example below and under <a href="https://www.w3.org/TR/owl2-overview/">OWL entailment</a> [[OWL2-Overview]], while `:superman` and `:clark` are inferred to denote one and the same person (because of the `owl:sameAs` relationship between them), the two embedded triples are not automatically considered to denote the same thing.</p>
+      <p>Similarly, in the example below and under <a href="https://www.w3.org/TR/owl2-overview/">OWL entailment</a> [[OWL2-Overview]], while `:superman` and `:clarkKent` are inferred to denote one and the same person (because of the `owl:sameAs` relationship between them), the two embedded triples are not automatically considered to denote the same thing.</p>
```

Comment:
To my mind, the ClarkKentIsSuperman example is more of a demonstration of the need for graph-level if not statement(triple or quad)-level permissions, for all statements, whether asserted, inferred, quoted, or otherwise.

JimmyOlsen, ClarkKent, Superman, Batman, BruceWayne, and a few others might be granted permission to know the truth of { ex:Superman owl:sameAs ex:ClarkKent}, and therefore be permitted to draw inferences from other statements about either of those entities -- but *most* are not granted that permission, and so should not be permitted to draw such inferences.

It's a important thing to consider -- just like it's important that members of the Rebel Alliance should be able to conceal a statement saying so from members of the Imperial Guard.  This may seem silly, but to my thinking, it flows directly from GDPR and similar rules, laws, etc.

(And yes, this is all tangential to RDF vs RDF-star vs RDF-plus-plus-plus -- but it should be a recognized concern throughout.)

Reply:
I don't think I agree because visibility and the ability to draw inferences to me seem too different to relate them in teh way you do, but, regardless, isn't this a topic on its own, outside the scope of this PR?